inference/core/workflows/core_steps/transformations/dynamic_crop/v1.py

```diff
@@ -67,6 +78,29 @@ class BlockManifest(WorkflowBlockManifest):
         examples=["$steps.my_object_detection_model.predictions"],
         validation_alias=AliasChoices("predictions", "detections"),
     )
+    mask_opacity: Union[
+        WorkflowParameterSelector(kind=[FLOAT_ZERO_TO_ONE_KIND]),
+        float,
+    ] = Field(
+        default=0.0,
+        le=1.0,
+        ge=0.0,
+        description="For instance segmentation, mask_opacity can be used to control background removal. "
+        "Opacity 1.0 removes the background, while 0.0 leaves the crop unchanged.",
+        json_schema_extra={"always_visible": True},
```

Comment:
Should this be always visible? Leaning towards no because it's not relevant for bounding boxes